assemblies/assembly_migrating-6.4-to-7.x/main.adoc

```diff
@@ -19,7 +19,7 @@ in {PRODUCT_OLD} to {PRODUCT} {PRODUCT_VERSION}. Red Hat provides migration tool
 
 .Prerequisites
 * {PRODUCT_OLD} is installed and contains artifacts that you want to migrate to {PRODUCT} {PRODUCT_VERSION}.
-* {PRODUCT} {PRODUCT_VERSION} is installed on a separate instance from {PRODUCT_OLD} (on {EAP} or other instance). For installation options, see {URL_PLANNING_INSTALL}[_{PLANNING_INSTALL}_].
+* A clean {PRODUCT} {PRODUCT_VERSION} installation that does not contain a `.niogit` folder is installed on a separate instance from {PRODUCT_OLD} (on {EAP} or other instance). If the {PRODUCT} {PRODUCT_VERSION} installation contains a `.niogit` folder, the installation will fail. For installation options, see {URL_PLANNING_INSTALL}[_{PLANNING_INSTALL}_]. 
```

Comment:
`is installed on a separate instance from {PRODUCT_OLD} (on {EAP} or other instance)` This sentence is little bit strange. What the instance means? Why instance from {PRODUCT_OLD}?

Reply:
Tomas, perhaps the confusion is with the variables? Here is the rendered PAM text:
A clean Red Hat Process Automation Manager 7.4 installation that does not contain a .niogit folder is installed on a separate instance from Red Hat JBoss BRMS 6.4 (on Red Hat JBoss EAP or other instance). If the Red Hat Process Automation Manager 7.4 installation contains a .niogit folder, the installation will fail. For installation options, see Planning a Red Hat Process Automation Manager installation.

Reply:
I still think It does not make sense. It sounds like RHPAM 7.4 installation is installed on BRMS 6.4.

I thinks we should have two installations:

1. Clean RHPAM 7.4 (without .niogit)
2. BRMS with niogit.

Because the migration tool is run on the 2. and then the migrated niogit is copied to the 1.
Is that correct?